concourse/steps/scan_sources.mako

```diff
@@ -129,4 +131,18 @@ github.compliance.report.create_or_update_github_issues(
 )
 % endif
 
+delivery_service_client = ccc.delivery.default_client_if_available()
+if delivery_service_client:
+    logger.info('Uploading result to delivery-service')
+    try:
+        checkmarx.util.upload_results_to_delivery_service(
```

Comment:
I think there is no need to be thus defensive. the data-upload happens as last step - so there is nothing bad happening if we let an error propagate.